src/mpyl/run_plan.py

```diff
@@ -115,7 +115,7 @@ def write_to_json_file(self):
             for execution in executions:
                 stages: list[dict[str, Union[str, bool]]] = run_plan.get(
                     execution.project.name, {}
-                ).get("stages", [])
+                ).get("plan", [])
```

Comment:
I don't understand this change, is this a find/replace gone wrong or am I missing something? 🤔